lib/workers/repository/update/branch/schedule.ts

```diff
@@ -13,6 +20,14 @@ const scheduleMappings: Record<string, string> = {
   monthly: 'before 5am on the first day of the month',
 };
 
+function parseCron(scheduleText: string): CronExpression | undefined {
+  try {
+    return parseExpression(scheduleText);
```

Comment:
I think we need to pass the timezone?

Reply:
Don't think it is necessary since we already apply timezone to the current date which we use for cron matches.
https://github.com/renovatebot/renovate/blob/2fbd472dbb665ddaa50193b5178346ec2a36fcfd/lib/workers/repository/update/branch/schedule.ts#L155

Reply:
but isn't the parsed cron timezone dependent? I would pass it just to be safe for future features.

Reply:
No. 
For example:
<details>
<summary>
With Timezone Asia/Kathmandu
</summary>

```ts

{
  _options: {
    tz: 'Asia/Kathmandu',
    currentDate: CronDate { _date: [DateTime] },
    nthDayOfWeek: 1
  },
  _utc: false,
  _tz: 'Asia/Kathmandu',
  _currentDate: CronDate {
    _date: DateTime {
      ts: 1689135538089,
      _zone: [IANAZone],
      loc: [Locale],
      invalid: null,
      weekData: null,
      c: [Object],
      o: 345,
      isLuxonDateTime: true
    }
  },
  _startDate: null,
  _endDate: null,
  _isIterator: false,
  _hasIterated: false,
  _nthDayOfWeek: 1,
  fields: {
    second: [ 0 ],
    minute: [
       0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11,
      12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
      24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35,
      36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47,
      48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59
    ],
    hour: [
       0,  1,  2,  3,  4,  5,  6,  7,
       8,  9, 10, 11, 12, 13, 14, 15,
      16, 17, 18, 19, 20, 21, 22, 23
    ],
    dayOfMonth: [
       1,  2,  3,  4,  5,  6,  7,  8,  9,
      10, 11, 12, 13, 14, 15, 16, 17, 18,
      19, 20, 21, 22, 23, 24, 25, 26, 27,
      28, 29, 30, 31
    ],
    month: [ 1, 3, 5, 7, 9, 11 ],
    dayOfWeek: [ 6 ]
  }
}
```

</details>


<details>
<summary>
Without Timezone
</summary>

```ts

{
  _options: { currentDate: CronDate { _date: [DateTime] }, nthDayOfWeek: 1 },
  _utc: false,
  _tz: undefined,
  _currentDate: CronDate {
    _date: DateTime {
      ts: 1689135718930,
      _zone: SystemZone {},
      loc: [Locale],
      invalid: null,
      weekData: null,
      c: [Object],
      o: -0,
      isLuxonDateTime: true
    }
  },
  _startDate: null,
  _endDate: null,
  _isIterator: false,
  _hasIterated: false,
  _nthDayOfWeek: 1,
  fields: {
    second: [ 0 ],
    minute: [
       0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11,
      12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
      24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35,
      36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47,
      48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59
    ],
    hour: [
       0,  1,  2,  3,  4,  5,  6,  7,
       8,  9, 10, 11, 12, 13, 14, 15,
      16, 17, 18, 19, 20, 21, 22, 23
    ],
    dayOfMonth: [
       1,  2,  3,  4,  5,  6,  7,  8,  9,
      10, 11, 12, 13, 14, 15, 16, 17, 18,
      19, 20, 21, 22, 23, 24, 25, 26, 27,
      28, 29, 30, 31
    ],
    month: [ 1, 3, 5, 7, 9, 11 ],
    dayOfWeek: [ 6 ]
  }
}
```

</details>

From the returned object we only need the `fields` object, which is timezone independent.